src/lib/PasswordHasher.js

```diff
@@ -14,24 +14,37 @@
 *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
 *    You can download the source here: https://github.com/spark/spark-server
+*
+* @flow
+*
 */
 
-var crypto = require('crypto');
-
-var PasswordHasher = function () {
+import crypto from 'crypto';
 
-};
+const HASH_DIGEST = 'sha1';
```

Comment:
Make sure it's the same hash as what we were using before.

We may change this to a stronger hashing algorithm if the hardware supports it.